.github/workflows/cross-platform.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
         steps:
             - name: Checkout sources
-              uses: actions/checkout@v2
+              uses: actions/checkout@v3
```

Comment:
are these changes relevant to the PR?

if not please extract and submit separately

Reply:
was trying to fix CI error but don't know how, it runs fine if I run the same commands locally (with nextest archive etc)

Reply:
please revert if not related and/or submit separately.